src/sage/modular/drinfeld_modform/tutorial.py

```diff
@@ -0,0 +1,230 @@
+r"""
+Introduction to Drinfeld modular forms
+======================================
+
+This tutorial outlines the definitions, the notations, and the
+implementation of Drinfeld modular forms in SageMath. We assume that the
+reader has basic knowledge of classical modular forms, as we will often
+make analogies to this setting. We also assume little knowledge of
+Drinfeld modules; for this topic, the interested reader can consult the
+SageMath reference manual: :ref:`Drinfeld modules`.
+
+.. RUBRIC:: Preliminary notations
+
+Let `q` be a prime power and let `A` be the ring of functions of
+`\mathbb{P}^1/\mathbb{F}_q` which are regular outside a closed point
+`\infty`. This ring is the polynomial ring `\mathbb{F}_q[T]`. We denote
+by `K := \mathbb{F}_q(T)` rational function field. We endow `K` with the
+`1/T`-adic valuation and let `K_{\infty} := \mathbb{F}_q((1/T))` be the
+completion of `K`. Next, we define `\mathbb{C}_{\infty}` to be the
+completion of an algebraic closure of `K_{\infty}`. Lastly, We denote
+by `\tau : x\mapsto x^q` the `q`-Frobenius.
+
+.. note::
+
+    The above construction of `\mathbb{C}_{\infty}` is the same as the
+    construction of `\mathbb{C}_p` in the case of `p`-adic numbers
+    (see :wikipedia:`P-adic_number#Algebraic_closure`).
+
+In SageMath, we create the rational function field by first creating a
+univariate polynomial ring over `\mathbb{F}_q` and, following this, by
+constructing its field of fractions::
+
+    sage: A = GF(3)['T']
+    sage: K.<T> = Frac(A)
+    sage: K
+    Fraction Field of Univariate Polynomial Ring in T over Finite Field of size 3
+    sage: K.base()  # returns A
+    Univariate Polynomial Ring in T over Finite Field of size 3
+
+.. RUBRIC:: Drinfeld period domain and action of `\mathrm{GL}_r(K_{\infty})`
+
+In the classical setting, the domain of any modular forms is the complex
```

Comment:
I would say
```suggestion
In the classical setting, the domain of any modular form is the complex
```
but I'm not sure.